onnxscript/tools/transformers_models/__init__.py

```diff
@@ -22,6 +27,38 @@ def has_transformers():
         return False
 
 
+def has_torch(version: str = ""):
+    """Tells if pytorch is installed and recent enough."""
+    import packaging.version as pv
```

Comment:
nit: Avoid unconventional abbreviations. I would just type the whole thing.

Reply:
I would also move the import out. We should have inline imports like this only in rare cases (e.g. torch/transformers etc. in command line utils)

Reply:
removed